src/components/blend-model.js

```diff
@@ -0,0 +1,40 @@
+var registerComponent = require('../core/component').registerComponent;

```

Comment:
I think we should include all of the hand controls components out of the box since it's important. They are prescriptive, but in the next release, we can focus on lazy-loading components and extract components out to be managed elsewhere. Then they will be readily available, but not actually part of the repo.